store/src/java/com/zimbra/cs/imap/ImapSearch.java

```diff
@@ -33,6 +36,8 @@ abstract class ImapSearch {
     protected abstract String toZimbraSearch(ImapFolder i4folder) throws ImapParseException;
     protected abstract ImapMessageSet evaluate(ImapFolder i4folder) throws ImapParseException;
     protected boolean requiresMODSEQ()  { return false; }
+    private static final String UID_SEARCH_PATTERN = "^1:(\\d+),(\\d+):\\*$";
```

Comment:
please add a comment for UID pattern that we going to ignore in the search

Reply:
Done. Added details of the pattern along with example